### ISSUE TITLE
Properties Editor - Modifiers - Geometry Nodes - Input sockets has double labels

### ISSUE THREAD

**Issue by Draise14** [Bforartists/Bforartists#3435]
![image](https://user-images.githubusercontent.com/35357409/222276586-41e1b9fc-6fc6-4596-91de-23f2b66cd03b.png)

Right now when you plug into the Group input node anything like a boolean, float, etc - you will see two labels. If you don't see two labels, you can unplug them and the residual input props will be double column. 

![bforartists_8Ww5ZAVzlk](https://user-images.githubusercontent.com/35357409/222276960-26adf07b-a041-43d8-b413-55401c5621cd.gif)

**Comment by ReinerBforartists** [comment]
I don't even know how to get there. But the one looks like a label, not a button.

**Comment by Draise14** [comment]
@iyadahmed this one has a bounty!

**Comment by iyadahmed** [comment]
Awesome will get to it asap ^*, I have to travel tomorrow though ^^

**Comment by Draise14** [comment]
Possible Clue:
`source/blender/modifiers/intern/MOD_nodes.cc`

**Comment by Draise14** [comment]
![imagen](https://user-images.githubusercontent.com/35357409/229323252-f22d8168-fa43-4829-9b72-252f39f29323.png)

Ok, this.. "fixes" it, but now the custom check box is centered, with the label.

**Comment by Draise14** [comment]
I think I can revert this issue and do the release, but making it float left will have to be done in another day. 

**Reason:**
- There are two states to the prop, and it's all very dynamic: if a socket is plugged in, it is a different check box. If it is not plugged into anything, it is another check box.
- I got one of the states floating left, but not the plugged in one... I have never done this before. 

So.. I need some ideas, but atleast I know where it's happening to a degree.

**Comment by ReinerBforartists** [comment]
One strategy is always to compare the file with Blender first.

We miss a few significant differences compared to Blender. Which could explain our trouble. Let me have a look how it looks after cleaning up.

![diff](https://user-images.githubusercontent.com/13919702/229337041-c5272af1-b260-4ad6-ace9-5cc5d48e9554.jpg)

**Comment by ReinerBforartists** [comment]
Great, throws an error now. So we miss another bit somewhere else too -.-

**Comment by ReinerBforartists** [comment]
The double label comes from a merge regression. To align left becomes now much more complicated. And resides behind a split layout. Which makes it hella complicated to sort out the props that we want at the left side. Because we cannot simply remove the split or align the split completely left. I tried ^^^

We loose the labels. And the boolean checkbox is still just in the second row of the split.

Let's revert to Blender default so that it does at least not look broken. And fix this issue at another day.

![splitdilemma](https://user-images.githubusercontent.com/13919702/229338391-8b2be6d1-2b82-4fd4-a2ca-3a7d6d6d0bbb.jpg)

![booleanleft](https://user-images.githubusercontent.com/13919702/229338394-ef279f8b-0b84-4d93-8106-d38316a037be.jpg)

**Comment by ReinerBforartists** [comment]
Postponing ...

**Comment by ReinerBforartists** [comment]
Ah, forgot to add the labels at the left side again.

The double label is fixed. And it is in real just one prop that needs special treatment. The boolean checkbox. Oh, and why don't we have a geometry node in the modifier? ^^

![boolean](https://user-images.githubusercontent.com/13919702/229340370-ed93e9c8-1c38-4f08-9fbd-20206a2fc035.jpg)

**Comment by Draise14** [comment]
Yeah. I guess revert is good enough for release for now.  

The geometry socket is inherent to the object. A bit like "execute" in a flow graph. It is "self".

**Comment by ReinerBforartists** [comment]
Yes. As told, it's just the boolean left. And i am not sure if we had this fixed before ^^

**Comment by ReinerBforartists** [comment]
Just as a node, i had a crack at this boolean, and failed. No idea how we could convince it to be at the left side. It is this split property here that is in the way ...

**Comment by Draise14** [comment]
I don't think this was ever fixed before. Indeed this is a tricky case. I have been thinking about this one - but I doubt I will be able to get it... 

1. Most of the types of rows as a list - and they do have a label and in another column the content. 
2. This one item in that list, the checkbox boolean, also has a label and content, but it's inversed. If it shows a label - it will show it twice. 
3. The checkbox draws the label of self outside of the list of props maybe....

**Comment by ReinerBforartists** [comment]
Sometimes you simply loose. We could mark this as won't fix ...

**Comment by ReinerBforartists** [comment]
This is as close as i can get. I have no idea why it does not align. It should. 

*sadcatnoise*

![stillnotleft](https://user-images.githubusercontent.com/13919702/233582265-10afe3eb-9e02-4f2b-9ad6-192b943d9453.jpg)

**Comment by Draise14** [comment]
I got it like this too... But it is not nominal. 

I assume it's how the labels and props are defined in columns in the row. The name of the boolean is not the label but the prop value (which happens to be the lable) like the numbers in a slider. The label is hidden already.

I have managed to show the label in the correct place (left column) but not hide the prop value (read label) so it looks like a double label. 

But in this case maybe a full float left checkbox out of the prop column might be not what we need anyway. The task is.. remove the checkbox prop readout of its own value...
